drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -65,6 +65,7 @@ struct aci_set_ble_addr {
 
 #define ACI_WRITE_SET_TX_POWER_LEVEL       BT_OP(BT_OGF_VS, 0xFC0F)
 #define ACI_HAL_WRITE_CONFIG_DATA	   BT_OP(BT_OGF_VS, 0xFC0C)
+#define ACI_HAL_STACK_RESET		   BT_OP(BT_OGF_VS, 0xFC3B)
```

Comment:
@erwango btw, I noticed that `drivers/bluetooth/hci/spi.c` also has these defines:https://github.com/zephyrproject-rtos/zephyr/blob/3cbfc505b357f80b6aca4cd2dc39c0f60a24c1b1/drivers/bluetooth/hci/spi.c#L106-L109

Perhaps they should be moved into a common header file?

It also seems like the defines here in `ipm_stm32wb.c` have slightly different values - I think the `0xFC..` part of the OCF is redundant, since that'll come from the OGF value when combined with it.